docs/software/python-cli/usage.mdx

```diff
@@ -14,7 +14,7 @@ The `meshtastic` command is not run within python but is a script run from your
 
 ## Viewing Serial Output
 
-The `--noproto` command in the Meshtastic Python CLI is used to disable the API and function merely as a "dumb serial terminal."  This mode of operation allows both the API and device functionalities to remain accessible for regular use, while simultaneously providing a window into the raw serial output. This feature can be particularly useful for debugging, development, or understanding the low-level communication between devices.
+The `--noproto` command in the Meshtastic Python CLI is used to disable the API and function merely as a "dumb serial terminal."  This mode of operation allows both the API and device functionalities to remain accessible for regular use, while simultaneously providing a window into the raw serial output. This feature can be particularly useful for debugging, development, or understanding the low-level communication between devices.Requires a physically cabled serial connection. It will connect but not display information over a network (`--host`) or Bluetooth (`--ble`) connection.
```

Comment:
One typo here - just need a space before `Requires` at the start. Why not just make the verbiage exactly the same between both places? `Depends on a physically cabled serial connection. It will connect but not display information over a network (`--host`) or Bluetooth (`--ble`) connection.`

Reply:
No particular reason. for the different verbiage. The index seemed like a spot for terseness. Consistent is probably better! Wilco.